proxy.h

```diff
@@ -590,44 +590,81 @@ proxy<F> make_proxy(T&& value) {
 
 // The following types and macros aim to simplify definition of dispatch and
 // facade types prior to C++26
-namespace helper {
+namespace details {
+
+template <class Args, class O>
+struct one_overload_matching_traits : inapplicable_traits {};
+template <class Args0, class R, class... Args1>
```

Comment:
Q: Looks like `Args0` is `Args` and `Args1` is `PackedArgs`?

Reply:
`Args0` is `Args`: correct.
`Args1` is `PackedArgs`: what is `PackedArgs`? `Args1` is deduced from the overload signature.